doc/developers/index.rst

```diff
@@ -111,7 +111,7 @@ then submit a "pull request" (PR):
 
 Finally, go to the web page of the your fork of the scikit-learn repo,
 and click 'Pull request' to send your changes to the maintainers for review.
-request. This will send an email to the committers, but might also send an
+This will send an email to the committers, but might also send an
```

Comment:
The sentence is kinda wrong. It would be great if you could fix it. doing a PR will not sent a mail to the mailing list. It should say "but you might also consider sending an email to the mailing list...."